### PR TITLE
Added health info to GPS ephemeris

### PR DESCRIPTION
**Pull request by mbavaro**
..and fixed the fit interval..

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0-97-g942e64d
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=rtklib-ci-builds&prefix=v1.0-97-g942e64d/281-linux/)
+ s3://rtklib-ci-builds/v1.0-97-g942e64d/281-linux

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0-97-g942e64d-dirty
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=rtklib-ci-builds&prefix=v1.0-97-g942e64d-dirty/281-osx/)
+ s3://rtklib-ci-builds/v1.0-97-g942e64d-dirty/281-osx

File: src/rcv/swiftnav.c [hunk 1]
```diff
@@ -486,8 +488,9 @@ static void decode_gpsnav_common_dep1(uint8_t *_pBuff, eph_t *_pEph) {
   _pEph->toes = U4(_pBuff + 4);
   uWeekE = U2(_pBuff + 8);
   _pEph->sva = uraindex(R8(_pBuff + 10)); /* URA index */
-  _pEph->fit = U4(_pBuff + 14) ? 0 : 4;
-  _pEph->flag = U1(_pBuff + 15);
+  _pEph->fit = U4(_pBuff + 18) / 3600;
```
Comment (Paakkit) [inline src/rcv/swiftnav.c]: This looks correct indeed.

File: src/rcv/swiftnav.c [hunk 2]
```diff
@@ -1220,8 +1241,7 @@ static void startfile(raw_t *raw) {
 /* end input file ------------------------------------------------------------*/
 static int endfile(raw_t *raw) {
   /* flush observation data buffer */
-  if (!flushobuf(raw))
-    return -2;
+  if (!flushobuf(raw)) return -2  ;
```
Comment (Paakkit) [inline src/rcv/swiftnav.c]: Some extra spaces

Comment (Paakkit) [reply src/rcv/swiftnav.c]: Never mind, only looked at first commit.

**Pull request merged**